prisma/schema.prisma

```diff
@@ -259,6 +259,7 @@ model mentor {
   isRedacted                   Boolean                 @default(false)
   lastTimeCheckedNotifications DateTime?               @default(dbgenerated("'1970-01-01 00:00:00'::timestamp without time zone")) @db.Timestamp(6)
   notificationPreferences      Json?                   @db.Json
+  lastLogin                    DateTime?               @default(now()) @db.Timestamp(6)
```

Comment:
Can we also make sure that prisma is updating/setting the createAt/updatedAt fields?

Reply:
`createdAt` is set by the `@default(now())` annotation afaik, but `updatedAt` is a TypeORM relic that's not supported by Prisma iirc. That's the reason why we introduced `lastLogin`, as `updatedAt` doesn't accurately represent the last time someone changed an object in the DB which caused trouble in the automated account redaction job.

Reply:
It seems like there is [an option](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#updatedat) in Prisma for this.
It would still be cool to keep the updatedAt, just to keep track of changes.

Reply:
Hmm indeed, but TypeORM is our "source of truth" right now and we pull the prisma schema from the database. So we'd have to manually add `@updatedAt` every time we pull the schema 🤔 @Jonasdoubleyou is that a feasible approach? or should we wait until we ditch TypeORM

Reply:
Okay, that makes a lot of sense. Let's skip it until we finally got rid of TypeORM 👍